Sources/Fuzzilli/Mutators/OperationMutator.swift

```diff
@@ -79,6 +79,8 @@ public class OperationMutator: BaseInstructionMutator {
             newOp = DeleteElement(index: b.genIndex())
         case let op as CallMethod:
             newOp = CallMethod(methodName: b.genMethodName(), numArguments: op.numArguments)
+        case let op as CallComputedMethod:
```

Comment:
This no longer mutates anything, so let's remove this (together with the `.isParametric` flag of the operation)